ppr-api/src/ppr_api/models/financing_statement.py

```diff
@@ -209,9 +209,18 @@ def party_json(self, party_type, registration_id):
             if party.party_type == party_type or \
                (party_type == Party.PartyTypes.DEBTOR_COMPANY.value and
                 party.party_type == Party.PartyTypes.DEBTOR_INDIVIDUAL.value):
-                if not self.current_view_json and party.registration_id == registration_id:
+                if party.registration_id == registration_id and \
```

Comment:
just wondering if you can add a few comments on this if block, I'm having trouble following it ;)

Reply:
Comments added.